dpgen2/exploration/task/lmp_spin_task_group.py

```diff
@@ -0,0 +1,112 @@
+import itertools
+import random
+from pathlib import (
+    Path,
+)
+from typing import (
+    List,
+    Optional,
+)
+
+from dpgen2.constants import (
+    lmp_conf_name,
+    lmp_input_name,
+    lmp_traj_name,
+    model_name_pattern,
+    plm_input_name,
+    plm_output_name,
+)
+
+from .conf_sampling_task_group import (
+    ConfSamplingTaskGroup,
+)
+from .lmp import (
+    make_lmp_input,
+)
+from .task import (
+    ExplorationTask,
+)
+
+
+class LmpSpinTaskGroup(ConfSamplingTaskGroup):
```

Comment:
It seems that this task group generate lammps spin md tasks from templates. 

I would recommend naming it as, e.g. `LmpSpinTemplateTaskGroup`,  and use input name like `"lmp-spin-template"`